src/app/components/routing/application-routing.component.ts

```diff
@@ -38,19 +38,21 @@ export class ApplicationRoutingComponent implements OnInit {
     const userAccess$ = combineLatest([userDetails$, bookingFeatureToggle$]);
 
     userAccess$.pipe(map(([userDetails, bookingFeatureToggle]) => {
-      if (this.router.url !== '/' ) {
+      if (this.router.url !== '/') {
         return;
       }
       if (bookingFeatureToggle && AppUtils.isBookableAndJudicialRole(userDetails)) {
         return this.router.navigate([ApplicationRoutingComponent.bookingUrl]);
       }
-      userDetails && userDetails.userInfo && userDetails.userInfo.roles &&
-      (userDetails.userInfo.roles.includes('caseworker-ia-iacjudge')
-      || userDetails.userInfo.roles.includes('caseworker-ia-caseofficer')
-      || userDetails.userInfo.roles.includes('caseworker-ia-admofficer')
-      || userDetails.userInfo.roles.includes('caseworker-civil')
-      || userDetails.userInfo.roles.includes('caseworker-privatelaw'))
-      ? this.router.navigate([ApplicationRoutingComponent.defaultWAPage]) : this.router.navigate([ApplicationRoutingComponent.defaultPage]);
+      userDetails && userDetails.userInfo && userDetails.userInfo.roles
```

Comment:
Please add Unit Tests for this change. Thanks

Reply:
@udaydenduluri33 tests are fixed. Can you please review again?